xapian-core/matcher/synonympostlist.h

```diff
@@ -58,6 +58,12 @@ class SynonymPostList : public PostList {
      */
     bool want_unique_terms;
 
+    /** Flag indicating whether the weighting object needs the max wdf of
+     *  terms.
+     *  FIXME : Not implemented
```

Comment:
I have set it to false, in the constructor of SynonymPostList. Is that right? I am not very clear about what a SynonymPostList is and how it will affect the functioning of the max_wdf statistic.